### PR TITLE
Fixed: mark revision relation as protected

### PR DESCRIPTION
**Pull request by disconnect821**
Fixes https://github.com/wagtail-nest/wagtail-ab-testing/issues/53

This is a counter PR for https://github.com/wagtail/wagtail/issues/10678

can only be merged after the successful merging of https://github.com/wagtail-nest/wagtail-ab-testing/pull/52

- [x] changes
- [x] test

### CODE REVIEW

**Comment by disconnect821** [comment]
PR is ready for review. 
@Stormheg  @salty-ivy
thanks a lot for the guidance.

File: wagtail_ab_testing/test/tests/test_abtest_model.py
Comment (salty-ivy) [inline wagtail_ab_testing/test/tests/test_abtest_model.py]: nit: this can be a single line statement
```python 
self.ab_test.variant_revision.delete()
```

**Comment by Stormheg** [comment]
Hi @disconnect821, this is still on my todo list to review. I'll get to it don't worry!

**Review by Stormheg — APPROVED** [review]
Thanks @disconnect821 - this looks perfect to me!

Since this depends on #52, I'll wait for that PR to be merged before merging this one.

**Pull request merged**
